frontend/src/redux/modules/results.js

```diff
@@ -1,10 +1,14 @@
 // @flow
-
+const UPDATE_RESULTS = 'results/UPDATE_RESULTS'
 const initialState = {
-  games: {
+  games: ''

```

Comment:
Small thing, but change `games: ''` to `games: {}`. It makes it clearer to see that games will be data, not a string. Also if we were to add more actions, one action might try to append a game to games, but would fail because games is initially this string.